src/main/java/io/lettuce/core/tracing/DefaultTracingTagsCustomizer.java

```diff
@@ -0,0 +1,30 @@
+package io.lettuce.core.tracing;
+
+import io.lettuce.core.protocol.CommandArgs;
+import io.lettuce.core.protocol.ProtocolKeyword;
+
+/**
+ * Default implementation of a {@link TracingTagsCustomizer}.
+ *
+ * It associates tags containing information about {@link CommandArgs}, errors or exceptions with a running {@link Tracer.Span}.
+ *
+ * @since 5.2
+ */
+public class DefaultTracingTagsCustomizer implements TracingTagsCustomizer {
+
+    @Override
+    public void handleCommandArgs(Tracer.Span span, ProtocolKeyword type, CommandArgs<?, ?> args) {
```

Comment:
I'm not sure if exposing the CommandArgs is the best way to do this.
Maybe using the getFirstEncodedKey() or something similar is more appropriate here.